bliss/conf/base_config.yaml

```diff
@@ -21,13 +21,13 @@ paths:
 
 simulator:
     _target_: bliss.simulator.simulated_dataset.SimulatedDataset
-    n_batches: 128
-    valid_n_batches: 10  # 256
+    n_batches: 1
+    valid_n_batches: 0  # 256
     fix_validation_set: true
-    num_workers: 32
+    num_workers: 0
     sdss_fields:  # Used in simulator for sampling backgrounds and PSFs for decoder
-        dir: ${paths.sdss}
-        bands: [2]
+        dir: /data/sdss
```

Comment:
Were these changed intentionally? We should at least keep `dir: ${paths.sdss}` since there's no guarantee a user will have a /data directory.

Reply:
That was unintentional, thanks for catching that!